packages/next/build/swc/options.js

```diff
@@ -65,7 +66,7 @@ function getBaseSWCOptions({
         decoratorMetadata: emitDecoratorMetadata,
         react: {
           importSource: nextConfig?.experimental?.emotion
```

Comment:
Just change this line is enough:  `importSource: jsConfig?.compilerOptions?.jsxImportSource ?? (nextConfig?.experimental?.emotion ? '@emotion/react' : 'react')`

Reply:
Updated in: https://github.com/vercel/next.js/pull/35963/commits/920768714724a4aa353e439bebbc852a1ebfd0a5